app/code/Magento/Cms/Ui/Component/Listing/Column/BlockActions.php

```diff
@@ -60,7 +63,7 @@ public function prepareDataSource(array $dataSource)
         if (isset($dataSource['data']['items'])) {
             foreach ($dataSource['data']['items'] as & $item) {
                 if (isset($item['block_id'])) {
-                    $title = $this->getEscaper()->escapeHtmlAttr($item['title']);
+                    $title = $this->escaper->escapeHtmlAttr($item['title']);
```

Comment:
@Morgy93 one more suggestion. I believe that use of `escapeHtml` is more correct, because the value is used in html text

Reply:
Hmm, indeed. So it matches with the BlockActions with the PageActions.

However, this seems to have been changed on purpose by https://github.com/magento/magento2/commit/aeb8c3dccc8457cee523487b5601412b4de3a4d7, but there is no explanation in the commit message why and the test changes don't explain it either.
Would be great if some Adobe employee could tell us what's the description of the Jira ticket MC-15476 says exactly to understand why this change was made...

Reply:
Usually, I'd say it is out of scope for this PR. 😅 But I am a fan of "leaving code better than it was before".

I changed it before I saw the response by hostep but would love to wait for the official answer by Adobe.

Reply:
Yeah, just for completeness sake, if you search the entire codebase, it seems like everywhere `escapeHtml` is used (see examples below), it's only in the BlockActions that `escapeHtmlAttr` is used, which is odd. So I'm very curious what ticket MC-15476's description will be ...

Examples:
- https://github.com/magento/magento2/blob/9e2eb996e4cf181c420f6801ded74b49428b546a/app/code/Magento/Cms/Ui/Component/Listing/Column/PageActions.php#L91
- https://github.com/magento/magento2/blob/9e2eb996e4cf181c420f6801ded74b49428b546a/app/code/Magento/Customer/Ui/Component/Listing/Column/GroupActions.php#L112-L115
- https://github.com/magento/magento2/blob/9e2eb996e4cf181c420f6801ded74b49428b546a/app/code/Magento/Sales/Ui/Component/Listing/Column/Address.php#L52
- https://github.com/magento/magento2/blob/9e2eb996e4cf181c420f6801ded74b49428b546a/app/code/Magento/Store/Ui/Component/Listing/Column/Store.php#L115-L117

Reply:
@hostep I just made the same research :) 
At the same time GroupActions looks very interesting because `$title` is escaped at line 89
https://github.com/magento/magento2/blob/9e2eb996e4cf181c420f6801ded74b49428b546a/app/code/Magento/Customer/Ui/Component/Listing/Column/GroupActions.php#L89
https://github.com/magento/magento2/blob/9e2eb996e4cf181c420f6801ded74b49428b546a/app/code/Magento/Customer/Ui/Component/Listing/Column/GroupActions.php#L112-L115
I can understand the changes as mass replace

Reply:
@engcom-Charlie @engcom-Bravo @sidolov  Could you please find MC-15476 ticket and help us understand the innermost thought of these changes

Reply:
Ow boy, it's being escaped twice there, we are finding quite some weirdness here 😆

Reply:
Hmm, feedback on Slack:
> Hey! That’s a tough one, unfortunately :disappointed:
> MC is an old Magento Jira project that was deprecated a few years ago, then we migrated to Adobe Jira and some of the info was lost during the migration

I think we should just ignore this escaping and proceed with the PR as-is. Since nobody knows why it happened, i think the escaping can be dealt with another time.

Reply:
Thanks @hostep 
Lets stay with current solution